spalp.module

```diff
@@ -59,3 +60,47 @@ function spalp_get_app_ids(FieldStorageConfig $definition, ContentEntityInterfac
   \Drupal::moduleHandler()->alter('spalp_app_ids', $ids);
   return array_combine($ids, $ids);
 }
+
+/**
+ * Implements hook_modules_installed().
+ */
+function spalp_modules_installed($modules) {
+  $module_list = system_rebuild_module_data();
+
+  foreach ($modules as $module) {
+    // Does the module have a dependency on spalp?
+    $dependencies = array_keys($module_list[$module]->requires);
+    if (in_array('spalp', $dependencies)) {
+      spalp_create_nodes($module);
+    }
+  }
+}
+
+/**
+ * Create applanding nodes in each language when a module is enabled.
+ *
+ * @param string $module
+ *   The machine name of the module being installed.
+ *
+ * @throws \Drupal\Core\Entity\EntityStorageException
+ */
+function spalp_create_nodes($module) {
```

Comment:
Might be better to create a service which contains all the helper methods. This way it allows any service / module / drush command to use your service to create the nodes. This would keep the approach in line with D8 OO patterns and best practices.

**See an example of service and class here:**
https://github.com/dakkusingh/flickr/blob/8.x-1.x/src/Service/Helpers.php
https://github.com/dakkusingh/flickr/blob/8.x-1.x/flickr.services.yml

Reply:
good idea, but let's do it after we've got an alpha release - I've created https://www.drupal.org/project/spalp/issues/2996135

Reply:
see https://github.com/dakkusingh/entity_usage_addons/commit/2d3272b618855a187cf5f863e4b2463b82b46d2d